src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -768,14 +786,113 @@ def set_up_service_binds(self):
             service_connectors_def_list, service_bindings_def_list = parse_service_bindings(self.cmd,
                                                                                             self.get_argument_service_bindings(),
                                                                                             self.get_argument_resource_group_name(),
-                                                                                            self.get_argument_name())
+                                                                                            self.get_argument_name(),
+                                                                                            self.get_argument_customized_keys())
             self.set_argument_service_connectors_def_list(service_connectors_def_list)
             unique_bindings = check_unique_bindings(self.cmd, service_connectors_def_list, service_bindings_def_list,
                                                     self.get_argument_resource_group_name(), self.get_argument_name())
             if not unique_bindings:
                 raise ValidationError("Binding names across managed and dev services should be unique.")
             safe_set(self.containerapp_def, "properties", "template", "serviceBinds", value=service_bindings_def_list)
 
+    def set_up_create_containerapp_yaml(self, name, file_name):
```

Comment:
do you support update with --yaml

Reply:
Yes, we support, customer need to input the whole `serviceBinds`